actors/migration/nv9/top.go

```diff
@@ -22,11 +25,28 @@ import (
 
 // Config parameterizes a state tree migration
 type Config struct {
```

Comment:
Hmm. I often prefer omitting defaults to force the caller to think about the values. Of course, Go doesn't help since it won't complain hard enough if a new field is added. I'm going to leave it now, but open to restoring the defaults in a follow-up. You might argue that we have a better idea about what good values are than integrators do, which is probably true.